packages/astro/src/@types/astro.ts

```diff
@@ -1094,6 +1094,29 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		inlineStylesheets?: 'always' | 'auto' | 'never';
+		/**
+		 * @docs
+		 * @name build.concurrency
+		 * @type {boolean}
+		 * @default `1`
+		 * @description
+		 * The number of pages to build in parallel. In most cases, you should stick with the default value of `1`,
+		 * and batch or cache long running tasks like fetch calls or data access to improve the overall rendering time.
+		 *
+		 * Use this option only if the refactors are not possible. If the number is set too high, the page rendering
+		 * may slow down due to insufficient memory resources and because JS is single-threaded.
+		 *
+		 * In the future, Astro may reuse this option to render pages concurrently with multiple threads.
+		 *
```

Comment:
```suggestion
```
In docs, we typically don't make guesses about the future since they don't tell what happens NOW and they are generally not helpful to someone building now.

Is this some kind of warning? Does this mean, "Whatever you set here, it might show up in future and have an effect you're not expecting?" Even if so, we would normally wait until that DOES happen, then state clearly, "If you have a value set for `build.concurrent`..." So I feel this is very safe to remove.

Reply:
Yeah this is more of a warning that the behaviour may change in the future for something else. It's something Astro can do in the future, just that it's not a very high priority thing to tackle now.

Realistically I don't think we could change the behaviour in a minor even if they work similarly, but I do want a kind of leeway for us to do so in the future. Would adding `@experimental` with this note be fine? I don't quite want to go the "create a new `experimental.*` config to use this feature" flow here since it's not really experimental and has potentially valid usecases.

Reply:
@ematipico this isn't resolved yet

Reply:
See my latest comment!